aks/charts/letsencrypt/values.yaml

```diff
@@ -4,44 +4,25 @@
 
 replicaCount: 1
 
-namespace: poc
+namespace: letsencrypt
 
 image:
-  repository: h2floh/aadloginsample #h2floh/letsencrypt 
+  repository: h2floh/letsencrypt 
   tag: latest
   pullPolicy: IfNotPresent
-  env:
-  - name: KeyVaultName
-    value: "securewebapp-flwagner"
-  - name: CertificateName
-    value: "LetsEncrypt"
-  - name: AzureConnectionString
-    valueFrom:
-          secretKeyRef:
-            name: azuresecret
-            key: AzureConnectionString
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
 service:
   type: LoadBalancer
-  loadBalancerIP: 52.231.24.232
   port: 443
   targetPort: 443
 
 ingress:
   enabled: false
-  annotations:
-    kubernetes.io/ingress.class: traefik
-  rules:
-    endpoint: 
-      host: vameetsaks.koreacentral.cloudapp.azure.com
-      paths:
-      - path: /
-        serviceName: aadlogin
-        servicePort: 443
+
```

Comment:
This is a question! Do we inject those values (for example, aks host url, IP address etc.) from outside of the values.yaml?

Reply:
We don't use the ingress controller therefore I removed all options except disabled. Maybe we should remove the ingress template and all ingress related variables?!

Reply:
Oh that sounds good! Could you update those parts as well? 😁

Reply:
I already deleted the ingress template, removed the associated values from values.yaml, and also deleted unnecessary yaml files.